api/nodes/serializers.py

```diff
@@ -154,9 +155,28 @@ def get_pointers_count(self, obj):
         return len(obj.nodes_pointer)
 
     def get_unread_comments_count(self, obj):
```

Comment:
Does this get computed for every node that gets serialized, or is this opt-in (e.g. via a query parameter, as with `?related_count=true`). Does this significantly impact response times? Would be good to benchmark this with projects with many comments.